app/controllers/hmis_controller.rb

```diff
@@ -0,0 +1,61 @@
+
+class HmisController < ApplicationController
+  # Permission notes: This is limited to people who can upload HMIS data
+  # and further limited to the data sources someone can edit.  This means
+  # the user must be granted explicit access to a data source and already
+  # has access to the source data.
+  before_action :require_can_upload_hud_zips!
+  before_action :set_item, only: [:show]
+  before_action :searched?, only: [:index]
```

Comment:
Why not just use the helper method?

Reply:
Ah, yes, the `if searched?` at the top of index means we no longer need the before_action.  Thanks.